README.adoc

```diff
@@ -41,8 +41,14 @@ image:https://dev.opera.com/extensions/branding-guidelines/addons_206x58_en@2x.p
  2. Open local or remote .ad, .adoc, .asc, .asciidoc files in Firefox
  3. Enjoy!
 
-WARNING: On Firefox, pages viewed directly from GitHub may not render properly. This is result of a https://bugzilla.mozilla.org/show_bug.cgi?id=1267027[Firefox bug] carrying over the restrictive content security policies (CSP) that GitHub sends to content inserted by the WebExtension.
-Please note that this issue can also occured on other websites with restictive CSP.
+[WARNING]
+====
+On Firefox, pages viewed directly from GitHub may not render properly. This is result of a https://bugzilla.mozilla.org/show_bug.cgi?id=1267027[Firefox bug] carrying over the restrictive content security policies (CSP) that GitHub sends to content inserted by the WebExtension.
```

Comment:
Ventilate this prose?

Reply:
as in, one sentence per line in the AsciiDoc document or think we should break down the second sentence ?

Reply:
I did both, let me know if it's better